bundle/InfoCollection/Form/Type/HoneypotType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\InfoCollection\Form\Type;
+
+use Ibexa\ContentForms\Form\Type\FieldType\TextLineFieldType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class HoneypotType extends AbstractType
```

Comment:
Ovaj type treba nasljedjivati TextType (ne kroz inheritance, nego kroz form type nasljedjivanje, `getParentType` metoda), da zadrzi sve njegove propertye i opcije fielda, jer u pozadini to je TextType

Reply:
Isto kao gore, s ovim kopiramo strukturu koju imaju ostali fieldovi generirani iz Contenta, gdje imamo `InfoCollectionFieldType` i pod njim npr. `TextLineType` koji extenda `TextType`.

Reply:
Svejedno ne mijenka cinjenicu da bi sam type trebao naslijediti `TextType` (kroz `getParent` metodu) iz Symfonya da zadrzi ponasanje samog text tipa iz Symfonya ako kojim slucajem on ima custom funkcionalnost, a sve po uputama na https://symfony.com/doc/current/form/create_custom_field_type.html

Reply:
Ne znam što to znači da bi trebao, cilje je pratiti strukturu ostalih polja i infocollection formi, a to s time ne bi mogli postići.

Reply:
`getParent() tells Symfony to take ChoiceType as a starting point`

Ovo znaci da ako npr Symfony na svoj TextType doda nekakvu opciju koja ima smisla na svim textualnim poljima, ta opcija se nece aplicirat ovdje.

Ili ako imas form ekstenziju koju isto zelis aplicirat na sve tekstualne inpute, neces moci jer onda ovaj field nije tehnicki TextType iz Symfony corea.

Reply:
Zanemari, brain fart s moje strane za `getParent`